### PR TITLE
Initial x86-x64-android support

### PR DESCRIPTION
**Pull request by mrcmunir**
Finish code on github and working fine tested under https://github.com/hrydgard/ppsspp/commit/afc212b2287caae451142fedd81bb6ced74b0cd4

No merge before  ffmpeg patches. 
https://github.com/hrydgard/ppsspp-ffmpeg/pull/38

### CODE REVIEW

File: android/jni/Application.mk [hunk 1]
```diff
@@ -1,5 +1,5 @@
 APP_STL := gnustl_static
 APP_PLATFORM := android-9
-APP_ABI := arm64-v8a armeabi-v7a x86
+APP_ABI := arm64-v8a armeabi-v7a x86 x86_64
```
Comment (unknownbrackets) [inline android/jni/Application.mk]: Poor Travis... oh well.

-[Unknown]

**Comment by hrydgard** [comment]
I have merged the ffmpeg change, please update the ffmpeg submodule in this pull request.

**Comment by maximu** [comment]
It is the first time I upgrade pull request submodule with git  if it is wrong tell me thanks.

**Comment by hrydgard** [comment]
It would have been enough to just upgrade the ffmpeg submodule - the lang one you are updating to a slightly old version. So either update them all to the latest or just update the ffmpeg one :)

**Comment by maximu** [comment]
Finished think you're all updated last wing :)

**Comment by Anuskuss** [comment]
A not unreleated question: Do we already have x64 support for iOS?

**Comment by maximu** [comment]
What? IOS Only supported arm architecture :s

**Comment by Anuskuss** [comment]
It's called AArch64, but you knew what I mean ;)

**Comment by unknownbrackets** [comment]
@Anuskuss There's no reason to believe @maximu even has an iOS device, so why are you asking him here?

I will await your pull request, @Anuskuss, adding AArch64 support for iOS.  Not that it will matter, since all iOS 9+ devices will crash with jit enabled anyway.

-[Unknown]

**Comment by Anuskuss** [comment]
Sorry, I'm very new to Github and don't want to bother anyone. I simply asked if you implent 64-bit support for Android, if there is (already) support for 64-bit in iOS. I think it would make a nice milestone (?) for 1.2.0.

**Comment by hrydgard** [comment]
@Anuskuss Apple's security got too good and we can't have JIT on 64-bit iOS anymore. No fun in working in such an environment, I prefer working on Android where we are allowed to do everything we need without jailbreaks.

**Pull request merged**